jerry-core/ecma/builtin-objects/ecma-builtin-math.c

```diff
@@ -152,7 +154,69 @@ ecma_builtin_math_object_max_min (bool is_max, /**< 'max' or 'min' operation */
 
   return ecma_make_number_value (result_num);
 } /* ecma_builtin_math_object_max_min */
+/**
+ * The Number.prototype object's 'Math.trunc' routine
+ *
+ * See also:
+ *          ECMA-262 v6, 20.2.2.35
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_number_t
```

Comment:
These all new functions/switch cases must be guarded with `JERRY_ES2015_BUILTNS` macro.